main.py

```
@@ -21,6 +21,12 @@
 # Define the categories for classifying emails.
 CATEGORIES = ["Orders", "Invoices", "Other"]
 
+try:
+    subject_email = os.getenv("GMAIL_SUBJECT_EMAIL")  
+except KeyError:
+    print("Error: GMAIL_SUBJECT_EMAIL environment variable not set.")
+    exit()
```

Pylint (3.3.7)

[refactor] Consider using 'sys.exit' instead

(R1722)